ChineseFilter/sections/specific.txt

```diff
@@ -1343,7 +1343,7 @@ wnacg.org##a[href^="https://l.tyrantdb.com/"][target="_blank"] > img
 6park.com##img[width="980"][height="90"]
 teepr.com##div[class^="mid-post-ad-"]
 ||159i.com/sxxd.js
-share.dmhy.org##div[align="center"] > a > img
+share.dmhy.org##div[id$="adv"]:has(a > img)
```

Comment:
As I see, the rule is fine. You can change it to this as there is another ad, but it's already blocked.
```adblock
share.dmhy.org##div[align="center"][id] > a > img
```

Reply:
ok, I have changed it to the proposed rule